### PR TITLE
Add cors test

### PR DESCRIPTION
**Pull request by juliovcruz**

### CODE REVIEW

File: handle/middlewares/cors_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+var allHeaders = []string{
+	"Vary",
+	"Access-Control-Allow-Origin",
+	"Access-Control-Allow-Methods",
+	"Access-Control-Allow-Headers",
+	"Access-Control-Allow-Credentials",
+	"Access-Control-Max-Age",
+}
+
+func assertHeaders(t *testing.T, resHeaders http.Header, expHeaders map[string]string) {
```
Comment (rodrigo-brito) [inline handle/middlewares/cors_test.go:33]: Maybe it is not necessary, this assertion will be used only in this file. What do you think?

File: handle/middlewares/cors_test.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+var GetCorsHeadersRequest = map[string]string {
+	"Origin":                        "http://foobar.com",
+	"Access-Control-Request-Method": "GET",
+	"Access-Control-Request-Headers": "Accept, Authorization, Content-Type",
+	"Access-Control-Allow-Methods": "*",
+}
+
+var GetCorsHeadersResponse = map[string]string {
```
Comment (rodrigo-brito) [inline handle/middlewares/cors_test.go:17]: Please, don't use global variable, it's a anti-pattern.

**Review by rodrigo-brito — CHANGES_REQUESTED** [review]
Hi @juliovcruz, nice job :clap: :clap:. Just a few comments.

**Comment by juliovcruz** [comment]
I agree with the changes, i believe it got better the code

**Comment by rodrigo-brito** [comment]
@juliovcruz thank you for the contribution

**Pull request merged**